test/e2e-common.sh

```diff
@@ -54,6 +54,13 @@ readonly PRE_INSTALL_V016="config/pre-install/v0.16.0"
 # Should deploy a Knative Monitoring as well
 readonly DEPLOY_KNATIVE_MONITORING="${DEPLOY_KNATIVE_MONITORING:-1}"
 
+TMP_DIR=$(mktemp -d -t ci-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX)
+readonly TMP_DIR
+readonly KNATIVE_DEFAULT_NAMESPACE="knative-eventing"
+
+# This the namespace used to install and test Knative Eventing.
+export TEST_EVENTING_NAMESPACE
+TEST_EVENTING_NAMESPACE=$(uuidgen | tr 'A-Z' 'a-z')
```

Comment:
OpenShift forces installation ins `knative-eventing`. I'd like to propose to change this line to:
```suggestion
TEST_EVENTING_NAMESPACE="${TEST_EVENTING_NAMESPACE:-"knative-eventing-"$(cat /dev/urandom \
  | tr -dc 'a-z0-9' | fold -w 10 | head -n 1)}"
```

Reply:
done